fe/fe-core/src/main/java/org/apache/doris/PaloFe.java

```diff
@@ -162,6 +166,28 @@ public static void start(String dorisHomeDir, String pidDir, String[] args) {
         }
     }
 
+    private static void checkAllPorts() throws IOException {
+        if (!NetUtils.isPortAvailable(FrontendOptions.getLocalHostAddress(), Config.edit_log_port,
```

Comment:
Make those suggesstions into const string ？

Reply:
Move to NetUtils